src/components/v5/common/ActionSidebar/partials/forms/ArbitraryTxsForm/consts.ts

```diff
@@ -0,0 +1,28 @@
+import { array, type InferType, object, string } from 'yup';
+
+import { ACTION_BASE_VALIDATION_SCHEMA } from '~v5/common/ActionSidebar/consts.ts';
+
+export const validationSchema = object()
+  .shape({
+    title: string()
+      .trim()
+      .required(() => 'Please enter a title.'),
+    decisionMethod: string().defined(),
+    transactions: array()
+      .of(
+        object()
+          .shape({
+            contract: string().defined(),
+            json: string().defined(),
+            method: string().defined(),
+            amount: string().defined(),
+            to: string().defined(),
```

Comment:
this is temporary, in next iterations we will update this fields to be dynamic (according to ABI from the contract)